bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/model/SomfyTahomaDeviceDefinition.java

```diff
@@ -25,19 +25,28 @@
 public class SomfyTahomaDeviceDefinition {
 
     private ArrayList<SomfyTahomaDeviceDefinitionCommand> commands = new ArrayList<>();
+    private ArrayList<SomfyTahomaDeviceDefinitionState> states = new ArrayList<>();
```

Comment:
```suggestion
    private List<SomfyTahomaDeviceDefinitionState> states = new ArrayList<>();
```

also in the line above

Reply:
ok